scripts/build/rollup.bin.config.js

```diff
@@ -11,7 +11,11 @@ export default Object.assign(baseConfig, {
   format: "cjs",
   banner: "#!/usr/bin/env node",
   plugins: [
-    replace({ "#!/usr/bin/env node": "" }),
+    replace({
+      "#!/usr/bin/env node": "",
+      // See comment in jest.config.js
+      "require('graceful-fs')": "require('fs')"
```

Comment:
I tried doing this with [`rollup-plugin-alias`](https://github.com/rollup/rollup-plugin-alias), but couldn't get it working.